src/modules/flow/iio/iio.json

```diff
@@ -822,6 +822,12 @@
              "name": "sampling_frequency"
          },
          {
+             "data_type": "string",
+             "default": null,
+             "description": "Channel name. If none, try to auto detect channel",
+             "name": "channel_name"
```

Comment:
@laykuanloon Please check the new added channel_id option, we can use it to read in_proximity2_raw/in_proximity3_raw